central_v2/public/locales/en/translation.json

```diff
@@ -41,6 +41,7 @@
     "test": "Test",
```

Comment:
I know we talked about two PRs for this one (and there were good reasons why we don't have them). In the future though, if we nest them, it is easier to review a smaller grouping of files. Mani has told me that around 15 files is a good max for a PR (obviously there's a lot of factors but it's just a rule of thumb). Useful way of looking at things I find